tests/lit-tests/max-vec.ispc

```diff
@@ -0,0 +1,51 @@
+// This test checks max functions for short vectors. They are now defined in standard library.
+
+// RUN: %{ispc} -O2 --vectorcall --target=avx512icl-x4 --emit-asm --x86-asm-syntax=intel %s -o - 2>&1 | FileCheck %s
```

Comment:
Is `vectorcall` essential for this test?

Reply:
Yes, otherwise on Windows it would be different asm.